common/src/main/java/com/hedera/block/common/utils/FileUtilities.java

```diff
@@ -62,21 +65,22 @@ public final class FileUtilities {
                             PosixFilePermission.OTHERS_EXECUTE));
 
     /**
-     * Log message template used when a path is not created because a file
-     * or folder already exists at the requested path.
+     * Log message template used when a path is not created because a file or folder already exists
+     * at the requested path.
      */
     private static final String PRE_EXISTING_FOLDER_MESSAGE =
             "Requested %s [%s] not created because %s already exists at %s";
 
     /**
-     * Create a new path (folder or file) if it does not exist.
-     * Any folders or files created will use default permissions.
+     * Create a new path (folder or file) if it does not exist. Any folders or files created will
+     * use default permissions.
      *
-     * @param toCreate valid, non-null instance of {@link Path} to be created
-     * @param logLevel valid, non-null instance of {@link System.Logger.Level} to use
+     * @param toCreate         valid, non-null instance of {@link Path} to be created
+     * @param logLevel         valid, non-null instance of {@link System.Logger.Level} to use
```

Comment:
Please, please, please do _not_ start this horrid pattern.  There is no reason to use aligned indentation here, and it's ridiculously costly as people have to constantly reformat lines for simple variable renames.

Reply:
✅ I reversed the formatting on the docs everywhere.